src/main/java/com/minecolonies/entity/ai/citizen/fisherman/Pond.java

```diff
@@ -72,15 +76,16 @@ private static boolean checkWater(IBlockAccess world, BlockPos water)
     private static boolean checkWaterPoolInDirectionXThenZ(IBlockAccess world, int x, int y, int z, int vector)
     {
         //Check 6 blocks in direction +/- x
-        for (int dx = x + 6 * vector; dx <= x + 6 * vector; dx++)
+        for (int dx = x + WATER_POOL_WIDTH_REQUIREMENT * vector; dx <= x + WATER_POOL_WIDTH_REQUIREMENT * vector; dx++)
         {
             if (world.getBlockState(new BlockPos(dx, y, z)).getBlock() != Blocks.water)
             {
                 return false;
             }
         }
         //Takes the middle x block and searches 3 water blocks to both sides
-        return checkWaterPoolInDirectionZ(world, x + 3 * vector, y, z, 1) && checkWaterPoolInDirectionZ(world, x + 3 * vector, y, z, -1);
+        return checkWaterPoolInDirectionZ(world, x + WATER_POOL_HEIGHT_REQUIREMENT * vector, y, z, 1) && checkWaterPoolInDirectionZ(world, x + WATER_POOL_HEIGHT_REQUIREMENT
+                                                                                                                                               * vector, y, z, -1);
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)